pro/src/pages/CollectiveOfferVisibility/CollectiveOfferEditionVisibility.tsx

```diff
@@ -60,46 +60,43 @@ const CollectiveOfferVisibility = ({
     notify.success(message)
   }
 
-  if (isLoading) {
-    return <Spinner />
-  }
-
-  if (error) {
-    return null
-  }
-
   return (
-    <CollectiveOfferLayout subTitle={offer.name} isTemplate={isTemplate}>
-      {isOfferInstitutionActive ? (
-        <CollectiveOfferVisibilityScreen
-          mode={offer.isVisibilityEditable ? Mode.EDITION : Mode.READ_ONLY}
-          patchInstitution={patchEducationalInstitutionAdapter}
-          initialValues={extractInitialVisibilityValues(
-            offer.institution,
-            offer.teacher
-          )}
-          onSuccess={onSuccess}
-          institutions={institutionsPayload.institutions}
-          isLoadingInstitutions={isLoading}
-          offer={offer}
-          reloadCollectiveOffer={reloadCollectiveOffer}
-        />
-      ) : (
-        <OldCollectiveOfferVisibility
-          mode={offer.isVisibilityEditable ? Mode.EDITION : Mode.READ_ONLY}
-          patchInstitution={patchEducationalInstitutionAdapter}
-          initialValues={extractInitialVisibilityValues(
-            offer.institution,
-            offer.teacher
+    <>
+      <Spinner isLoading={isLoading} />
```

Comment:
```suggestion
      <Spinner isLoading={isLoading && !error} />
```
Sinon le loading sera encore affiché même en cas d'erreur.

Reply:
La proposition vaut aussi pour ceux plus haut aussi, et ceux d'après.